taco_parser/src/xml_parser_tests.rs

```diff
@@ -90,13 +94,13 @@ mod xml_parser_tests {
         println!("{:#?}", result);
     }
 
-    // #[test]
-    // fn test_process_taco_data_reactif() {
-    //     let folder = "/home/jl/Documents/reaktif_taco/POIs/".to_string();
-    //     let xml_file = "General__Tequatl.xml".to_string();
-    //     let result = process_taco_data(folder, xml_file);
-    //     println!("{:#?}", result);
-    // }
+    #[test]
+    fn test_process_taco_data_reactif() {
+        let folder = "/home/jl/Documents/taco_packs/teh_trails".to_string();
```

Comment:
This looks like an absolute path to a directory on your own system. I must have missed it before or ignored it because it was in a comment. this should be a relative path so that it works on other computers.

Reply:
Commented out, good catch.